src/test/resources/mappings/inventory_items.json

```diff
@@ -0,0 +1,30 @@
+{
+  "mappings": [
+    {
+      "request": {
+        "method": "GET",
+        "url": "/inventory/items?query=barcode==645398607547"
```

Comment:
x-okapi-token/x-okapi-tenant will be useful here just to be sure that correct values were populated during FeignClient processing.